EmailNotification/utils.ts

```diff
@@ -114,11 +114,24 @@ export const truncateMarkdown = (plainText: string): string =>
     : plainText.substring(0, MAX_CHARACTER_FOR_BODY_MAIL);
 
 export const removeLinks = (text: string): string =>
-  text.replace(/(?:https?|ftp):\/\/[\n\S]+/g, "");
+  text.replace(/(?:https?|ftp):\/\/[\n\S]+/g, "[link rimosso]");
```

Comment:
what if there's something like `xyz://anything.anything`? I'd suggest to find a regex that matches everything before a `://`

Reply:
[935a00a](https://github.com/pagopa/io-functions-services/pull/297/commits/935a00aace0fb262f6a646020b13df7f9b2040fd)